packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal-body.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Tabs, ThemedScrollbars } from '@deriv/components';
+import { isDesktop } from '@deriv/shared';
+import { getCashierOptions } from './provider';
+
+type TCashierWalletModalBodyProps = {
+    active_tab_index: number;
+    is_dark: boolean;
+    is_demo: boolean;
+    setActiveTabIndex: (index: number) => void;
+};
+
+const CashierWalletModalBody = React.forwardRef<HTMLDivElement, TCashierWalletModalBodyProps>(
+    ({ active_tab_index, is_dark, is_demo, setActiveTabIndex }, ref) => {
+        const getDemoTabsColor = () => {
+            if (is_demo) {
+                return is_dark ? '#333333' : '#C2C2C2';
```

Comment:
you can define a variable in the theme.scss file and cover both `light` and `dark` mode there instead of passing it here